src/java.base/share/classes/java/lang/Class.java

```diff
@@ -633,6 +633,7 @@ private static Class<?> forName(Module module, String name, Class<?> caller) {
      * @see #asValueType()
      * @since Valhalla
      */
+    @PreviewFeature(feature = PreviewFeature.Feature.VALUE_OBJECTS)
```

Comment:
I only expect `@PreviewFeature` on public APIs.   No need to mark the package private methods as they are implementation details.

Reply:
Removed. (That API may come back with "primitive" classes).